pkgs/development/tools/verible/default.nix

```
@@ -45,14 +49,56 @@ buildBazelPackage rec {
   ];
 
   postPatch = ''
-    patchShebangs bazel/build-version.py \
+    patchShebangs\
+      kythe-browse.sh \
+      verilog/tools/diff/diff_obfuscate_mismatch_test.sh \
+      verilog/tools/diff/diff_format_match_test.sh \
+      verilog/tools/diff/diff_format_mismatch_test.sh \
+      verilog/tools/diff/diff_format_lex_error_test.sh \
+      verilog/tools/diff/diff_user_errors_test.sh \
+      verilog/tools/diff/diff_obfuscate_match_test.sh \
```

```suggestion
      verilog/tools/diff/ \
```
It is also acceptable to patch directories we should just avoid . and many files.

---

It would probably also make sense to sort all these files alphabetically, so that it is easier to see which directories can be collapsed. `verilog/tools/formatter/` also looks like a good candidate to directory-apply.